app/ext_example/Makefile

```diff
@@ -179,13 +181,18 @@ test-thirdparty: test-%: ${CLI} ${TARGET}
 	@cmp /tmp/zsvext-$@.out test/expected/zsvext-$@.out && ${TEST_PASS} || ${TEST_FAIL}
 
 clean:
-	@rm -f ${TARGET} /tmp/zsvext-test*.out
+	@rm -f ${TARGET} ${TARGET_SHEET} /tmp/zsvext-test*.out
 
 ${BUILD_DIR}/objs/%.o : ${THIS_LIB_BASE}/src/%.c ${PARSER_DEPS}
 	${MAKE} -C ${THIS_LIB_BASE}/src CONFIGFILE=${CONFIGFILEPATH} DEBUG=${DEBUG} WIN=${WIN} $@
 
-${TARGET}: my_extension.c ${UTILS}
+
+YAJL_SRC_DIR=${THIS_MAKEFILE_DIR}/../external/yajl
+YAJL_INCLUDE=-I${YAJL_SRC_DIR}/build/yajl-2.1.1/include
+YAJL_HELPER_INCLUDE=-I${THIS_MAKEFILE_DIR}/../external/yajl_helper
+${TARGET_SHEET}: LIBS="../external/sqlite3/sqlite3.c" -lzsv -lzsvutil -L${PREFIX}/lib
```

Comment:
@liquidaty If I do a full build then I still don't seem to have the zsvutil library and it is mentioned as a feature coming soon, is this from another branch?

Reply:
@richiejp if you do a full build it should make that but if not, we will look into it and in the meantime pls try:
`make -C app install-util-lib`

Reply:
@liquidaty starting from a clean environment and doing `make build -j16 DEBUG=1` then `make app/ext_example all -j16 DEBUG=1` does not work. However `make -C app install-util-lib -j16 DEBUG=1` did allow the example to be built as suggested, thanks!